IAB Workshop.md

```diff
@@ -51,17 +52,18 @@ Regarding operator deployments, discussion topics will include, but are not limi
 # Laying a path for the future of Network Management
 
 Network topologies have become significantly more complex since the
-2002 IAB workshop, as have router and host technologies. 
-Assuming this trend will continue, this workshop's ambition is
+2002 IAB workshop, as have the router and host technologies. 
+Assuming this trend will continue, this workshop's incentive is
 to lay a new directional foundation for the continued
 future of network management protocols, gathering new input on 
 what new issues network operators and network management implementers are facing.
 
 The candidate items for discussion include, but are not limited to:
 
-- Tooling, opensource, experimentation, system integration
+- Tooling, opensource, experimentation, proof of concept, multi-vendor interoperability test (e.g., EANTC),
+  system integration
 
-- Data consistency to support richer observability (Data & Knowledge)
+- Data consistency to support richer full stack observability (Data & Knowledge)
```

Comment:
```suggestion
- Data consistency to support richer observability (Data & Knowledge)
```

Not sure "full stack" will understood. I'm having troubles myself with parsing its meaning here.